src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Crypto.cs

```diff
@@ -87,12 +87,12 @@ internal static int BioTell(SafeBioHandle bio)
 
         internal static string? GetX509RootStorePath()
         {
-            return Marshal.PtrToStringAnsi(GetX509RootStorePath_private());
+            return Marshal.PtrToStringAnsi(GetX509RootStorePath_private())!;
```

Comment:
Why do you need these two changes? `GetX509RootStorePath` and `GetX509RootStoreFile` return `string?`.

Reply:
Looks like I added it initially and then forgot to remove. Thanks.